lib/srv/termmanager.go

```diff
@@ -68,7 +68,7 @@ func NewTermManager() *TermManager {
 		writers:           make(map[string]io.Writer),
 		readerState:       make(map[string]bool),
 		closed:            make(chan struct{}),
-		readStateUpdate:   sync.NewCond(&sync.Mutex{}),
+		readStateUpdate:   make(chan bool, 1),
```

Comment:
What is the purpose of the buffered channel? In the simple case, it may prevent `On()` or `Off()` from blocking, but two consecutive calls to these may still cause blocking. If `TermManager.Read()` is being invoked in a tight-ish loop, it probably won't matter, but then we also should not need the buffered channel.

Reply:
Yeah, I can see in `lib/srv.session` that `On()` is called when a session is launched before any parties are added, so it will never get past that point without a buffered channel or moving the call to `On()` to after the party is added (or doing `On()` in a goroutine).